### DISCUSSION TITLE
Add Prop to prevent delelecting option in Listbox

### DISCUSSION THREAD

**Discussion by katerlouis** [primefaces/community#1668] [PrimeVue]
Currently, when you click on the currently selected Item in Listbox, the item gets deselected and the modeled value is set to null. It would be great to have an option to prevent that.

**Comment by CARodriguezG** [comment]
Any update about this? It would be great indeed to have a `mandatory` prop to avoid setting the model to `null`.

By the meantime, I have published a package with a Vue directive for getting a similar behaviour. The package with instructions can be [found here](https://www.npmjs.com/package/@carodriguezg/primevue-mandatory).

Greetings!

**Comment by erdemsusam** [comment]
Is there any plan on implementing this behavior?
